src/client/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function Footer() {
+
+    return (
+        <footer>
+        <div className="footer">
+                    <ul className="footer-container">
+                        <li>Meal Sharing App by Darinka G.M.</li>
+                    </ul>
+            </div>
+    </footer>
+    )
```

Comment:
Indentation looks a bit weird... if you try to format it with prettier this would be fixed.